src/engine/blocks.js

```diff
@@ -298,7 +298,7 @@ class Blocks {
             if (!block.fields[args.name]) return;
             if (args.name === 'VARIABLE') {
                 // Get variable name using the id in args.value.
-                const variable = optRuntime.getEditingTarget().lookupVariableById(args.value);
```

Comment:
Was this a bug originally? I don't see why you would pass the name of the variable to fetch the variable, since the ID never matched the name